packages/core/package.json

```diff
@@ -69,4 +69,4 @@
       ]
     ]
   }
-}
+}
```

Comment:
minor: we need to keep newline in the files

Reply:
oh...ok